spec/models/ingredient_spec.rb

```diff
@@ -12,4 +12,12 @@
       it { should have_many(:recipes).through(:recipe_ingredients) }
    end
 
+   it "can alphabetize ingredients by name" do
+      bread = Ingredient.create!(name: "bread", cost: 4)
+      apple = Ingredient.create!(name: "apple", cost: 2)
+      clove = Ingredient.create!(name: "clove", cost: 8)
+
+      expect(Ingredient.all.alphabetical_name).to eq([apple, bread, clove])
```

Comment:
We don't need to have the `.all` here. You will just have the `.alphabetical_name` method called.